### PR TITLE
Documentation for K2 node management and services

### PR DESCRIPTION
**Pull request by guineveresaenger**
This should cover all documentation for K2 in regards to node management. This is meant to cover issue #230 .

We should make sure to document upcoming changes to k2cli as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,56 +396,55 @@ clusters:
 ```
 - etcd settings (beyond machine type)
 
-### Things that can be changed (sometimes with some manual intervention)
+*Warning*
+Repeatedly running `./up.sh` or `./update.sh` can result in etcd nodes being destroyed and regenerated automatically, which will break your cluster's control plane. This is because in the K2 generated config file, the CoreOS version is set to `current`. When CoreOS releases a new version, Terraform will register this change and destroy and re-create etcd nodes. We are working on fixing this, but please be aware that this is a possible issue.
```
Comment (davidewatson) [inline README.md]: This behavior is destined to cause problems. Maybe we should default to the current latest `1353.8.0` and forgo this warning? I had to make a similar change to the original kraken...

**Comment by davidewatson** [comment]
LGTM. We need to structure some of the documentation, and possibly some of the flags (cf. https://github.com/samsung-cnct/k2/issues/508), but those items should not hold up this PR, IMO. The faster we merge code, the fewer merge conflicts we will have...

File: README.md (hunk 1 above)
Comment (guineveresaenger) [inline README.md]: @davidewatson : @coffeepac and I talked about this on Friday. We have plans for this, though I couldn't point you at the specific issues right now. As far as I understand, quickfix is to place etdc in its own Terraform folder, which will freeze its CoreOS version. Longterm fix is to make sure etcd gets updates safely.

**Comment by guineveresaenger** [comment]
Could someone approve and merge?

File: README.md (hunk 1 above)
Comment (coffeepac) [inline README.md]: I'll open a fresh issue for this.  Its a good discussion to have.

**Pull request merged**